.github/workflows/test.yml

```diff
@@ -4,10 +4,10 @@ on: [push, pull_request]
 
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.13.x, 1.14.x, 1.15.x]
+        go-version: [oldstable, stable]
```

Comment:
These aliases now exist: https://github.com/actions/setup-go?tab=readme-ov-file#using-stableoldstable-aliases

Reply:
Ignore that, it seems it didn’t work for use these aliases. I reverted it.